components/buttons/ButtonRounded.tsx

```diff
@@ -6,17 +6,19 @@ type ButtonRoundedPropTypes = {
   type: 'button' | 'submit' | 'reset';
   isDisabled?: boolean;
   clickHandler?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  width?: string;
 };
 
 const ButtonRounded: React.FC<ButtonRoundedPropTypes> = ({
   children,
   clickHandler,
   isDisabled,
   type,
+  width,
 }) => {
   return (
     <button
-      className='button button-rounded disabled:bg-primaryGray'
+      className={`button button-rounded disabled:bg-primaryGray ${width || ''}`}
```

Comment:
I don't think we need the `||` here 🤔  it's just cleaner if we don't add an extra evaluation